axelor-cash-management/src/main/java/com/axelor/apps/cash/management/service/ForecastRecapServiceImpl.java

```diff
@@ -237,7 +237,7 @@ protected void populateWithForecastLineType(
             .bind("toDate", forecastRecap.getToDate())
             .bind("statusSelectList", statusSelectList)
             .bind(
-                "bankDetailsSet",
+                "bankDetails",
                 CollectionUtils.isEmpty(forecastRecap.getBankDetailsSet())
                     ? null
                     : forecastRecap.getBankDetailsSet())
```

Comment:
I'm not sure this will work because in getFilter() method, there is : 
`AND (:bankDetails IS NULL OR self.companyBankDetails = :bankDetails)`
So the parameter bankDetails can't be a Set, it must be a single bankDetails

So I think this query must be changed to allow to use the forecastRecap.getBankDetailsSet()